services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from typing import List, Optional
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie=movie,
+                                cinema_hall=cinema_hall)
+
+
+def get_movies_sessions(
+        session_date: Optional[datetime.date] = None
+) -> List[MovieSession]:
+    if session_date is None:
+        return (MovieSession.objects.all().
+                values_list("show_time", "movie__title"))
+    else:
+        return (MovieSession.objects.
+                filter(show_time__date=session_date).
+                values_list("show_time", "movie__title"))
```

Comment:
Similar to the previous comment, if the intention is to return `MovieSession` objects, consider removing `values_list` to return the queryset directly.